### ISSUE TITLE
[Bug report] Aria Hidden attribute of OutboundLink Icon is misspelled

### ISSUE THREAD

**Issue by DerZade** [vuepress/vuepress-next#427]
### Description
The aria hidden attribute of all the icon next to all outbound links does not work. This is probably because it is misspelled. It should be `aria-hidden`, but is `ariaHidden`.

https://github.com/vuepress/vuepress-next/blob/05246d1001a3728205d018c8b1dc647670bd258d/packages/%40vuepress/client/src/components/OutboundLink.ts#L10

### Steps to reproduce

- Open screen reader of you choice.
- Open VuePress v2 docs (or any other docs built with VuePress v2)
- Use screen reader to read out GitHub link within the navbar
- Screen reader reads _"GitHub; Image; open in new window"_

Reproduction link / repo:  https://v2.vuepress.vuejs.org/

### Expected behavior

When using a screenreader the icon of outbound links should be skipped, like it is in VuePress v1.

### Screenshots
not related

### Environment info
not related
